setup.env

```diff
@@ -7,7 +7,7 @@
 ############################################################
 
 # Fix expired letsencrypt root certificate https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/
-sudo apt-get update; sudo apt-get install -y libgnutls30
+sudo apt-get update; sudo apt-get install -y libgnutls30 pxz
```

Comment:
I think it is better to add this to a Dockerfile and switch from ONL to dent project builder

Reply:
You are correct, this is a quick response to the security issue in gnutls, I was also experimenting with pxz to see if it helped speed up the build.  I will remove them both.  I need to submit the 1.7 Dockerfile which updates this.  I will remove this from the PR.